src/Service/EnforcementManager.php

```diff
@@ -266,4 +270,23 @@ protected function isEnabled(): bool
 
         return true;
     }
+
+    protected function isUserInMFAEnabledGroup(Member $member)
```

Comment:
```suggestion
    protected function isUserInMFAEnabledGroup(Member $member): bool
```

Someone might be able to correct me here but I believe we support a minimum of PHP7 in this repo now. We should add proper type hints to new API.

Can you please add a doc-block in a similar fashion to the other methods (with a similar purpose) in this class? Something like what you've got in this PR description, but not written as A/Cs would be nice 🙂